src/modules/system/System.cpp

```diff
@@ -53,13 +53,43 @@
 #endif
 #endif
 
+#if defined(LOVE_LINUX)
+#include <unordered_set>
+
+static std::unordered_set<pid_t>& getSpawnedPids()
+{
+	static std::unordered_set<pid_t> spawnedPids;
+	return spawnedPids;
+}
+
+static void sigchldHandler(int signo, siginfo_t* info, void* /*context*/)
```

Comment:
I'm very unfamiliar with these APIs so this might be a silly question: is it possible for this callback to be called on other threads, or is it guaranteed to only be called on the main thread?

If it can be called on other threads there'll need to be some extra thread safety (maybe a mutex) when modifying `spawnedPids`.